src/functions.php

```diff
@@ -10,12 +10,12 @@ function generateArtistHtml(array $artists): string {
     && array_key_exists('placeOfBirth', $artists)) {
         return '<h2>' . $artists['name'] . '</h2>' . 
         '<p class="years">' . $artists['yearsLived'] . '</p>' .
-        '<img alt="an image of artwork by ' . $artists['name'] . '" class="image" src=' . $artists['image'] . '>' .
+        '<img class="image" src="images/' . $artists['image'] . ' "alt="an image of artwork by ' . $artists['name'] . '">' .
```

Comment:
sorry I didn't spot that one on the first review but I don't think you need an image class - it currently looks like you could just style your images with a `img {}` selector.